frontend/src/scenes/session-recordings/templates/SessionRecordingTemplates.tsx

```diff
@@ -157,7 +157,9 @@ const RecordingTemplateCard = (props: RecordingTemplateCardProps): JSX.Element =
             closeable={variablesVisible}
             onClose={hideVariables}
             focused={variablesVisible}
-            data-attr={`session-replay-template-${props.category}-${props.template.key}`}
+            data-attr="session-replay-template"
+            data-ph-capture-attribute-category={props.category}
+            data-ph-capture-attribute-template={props.template.key}
```

Comment:
these two add properties to the autocaptured event so that we can graph usage by category or template